app/serializers/user_serializer.rb

```diff
@@ -145,18 +145,8 @@ def website
 
   def website_name
     website_host = URI(website.to_s).host rescue nil
-    discourse_host = Discourse.current_hostname
     return if website_host.nil?
-    if website_host == discourse_host
-      # example.com == example.com
-      website_host + URI(website.to_s).path
-    elsif (website_host.split('.').length == discourse_host.split('.').length) && discourse_host.split('.').length > 2
-      # www.example.com == forum.example.com
-      website_host.split('.')[1..-1].join('.') == discourse_host.split('.')[1..-1].join('.') ? website_host + URI(website.to_s).path : website_host
-    else
-      # example.com == forum.example.com
-      discourse_host.ends_with?("." << website_host) ? website_host + URI(website.to_s).path : website_host
-    end
+    website_host + URI(website.to_s).path
```

Comment:
no rescue here for URI(website.to_s) this means that bad URI somewhere could just explode the serializer